Bluetooth module/sample.py

```diff
@@ -0,0 +1,42 @@
+
+from bluetooth import *
+import sys
+
+if sys.version < '3':
```

Comment:
This sys.version is connected with Raspberry model? Or it's something different?